README.md

```diff
@@ -3,18 +3,30 @@
 tlock gives you time based encryption and decryption capabilities by relying on a [drand](https://drand.love/) threshold network.  
 It's also a Go library, which is used to implement the `tle` command line tool enabling anybody to leverage timelock encryption.
 
-Our timelock encryption system relies on an unchained drand network. Currently, the only publicly available one is the League of Entropy Testnet.
-However, it should soon also be available on the League's Mainnet.
+Our timelock encryption system relies on an "[unchained drand network](https://drand.love/blog/2022/02/21/multi-frequency-support-and-timelock-encryption-capabilities/)".
 
-Working endpoints to access it are, for now:
+Working endpoints to access it are, on mainnet:
+- https://api.drand.sh/ (US)
+- https://api2.drand.sh/ (EU)
+- https://api3.drand.sh/ (Asia)
+- https://drand.cloudflare.com/ (load-balanced across regions)
+
+On mainnet, only one chainhash supports timelock encryption:
+- running with a 3 seconds frequency with signatures on G1: `dbd506d6ef76e5f386f41c651dcb808c5bcbd75471cc4eafa3f4df7ad4e4c493`
+
+This is a production-ready network with high-availability guarantees. It is considered fully secure by the drand team 
```

Comment:
Could mention here that it's been audited. Kudelsk put out a [timelock-encrypted disclosure blog](https://research.kudelskisecurity.com/2023/01/09/announcing-a-timelocked-responsible-disclosure/) ... but it will only be decryptable using the `testnet-unchained-3s` on the old scheme x( we could maybe release our own blog post with the findings and our remediation?

Reply:
Let's do that later once we have the final version of the report 👍🏻